packages/cfd/src/Constants/mt5-migration-modal-content.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+
+const getMigrationModalDetails = (to_account: string) => [
```

Comment:
```suggestion
type TMigrationModalDetails = NonNullable<
    ReturnType<typeof useMT5SVGEligibleToMigrate>
>['eligible_account_to_migrate_label'];

const getMigrationModalDetails = (to_account: TMigrationModalDetails) => [
```
The next person working on this wont know what is `to_account` and where its coming from. Eventhough its a string or a boolean, we dont need to waste time figuring out where its originating from. Incase something got messed up in the top level, typescript will save us here 💯

Reply:
Also also a neat trick if there is any values coming from the hook or api is undefined for example. We can do it like this 
```
Exclude<NonNullable<
    ReturnType<typeof useMT5SVGEligibleToMigrate>
>['eligible_account_to_migrate_label'], undefined>
```
By this way we dont need to have like `to_account || 'fallback'`. 😸